Automattic/jetpack
# Add a unified search component.

This adds a new component that will include all tabs and pass search terms into them. That will allow us to get rid of a dedicated search view and use existing ones instead.

#### Changes proposed in this Pull Request:
- Adds searchable settings view.
- Changes the composing view inside the Writing tab to respect search terms.

#### Testing instructions:
- Try to search for something related to markdown and after-the-deadline modules.
- Observe the respective configuration settings

#### What else to look at
- [x] Make sure that when you try to search, and there's no search term yet, settings are not empty.
